phoenicis-engines/src/main/java/org/phoenicis/engines/Verb.java

```diff
@@ -0,0 +1,13 @@
+package org.phoenicis.engines;
+
+/**
+ * interface which must be implemented by all Verbs in Javascript
+ */
+public interface Verb {
```

Comment:
What about a unique name?
How does Phoenicis remember, whether a verb has been installed previously or not?

Reply:
It won't. Check @qparis comment in https://github.com/PhoenicisOrg/scripts/issues/534.

Reply:
Yes, but I think Phoenicis should remember which verbs were installed on a container.
Otherwise, the user can't track what he did to his container.

Reply:
So some kind of database? I guess we can consider that. However, we should do that later otherwise this PR will blow up too much.

Reply:
Yes, but I think in our case a simple json file per container, containing a list of verb ids/names would be enough.
I agree we can do this in a later PR.